presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -57,6 +57,8 @@
 #include "velox/dwio/parquet/RegisterParquetReader.h" // @manual
 #endif
 
+DECLARE_bool(velox_memory_leak_check_enabled);
```

Comment:
Let's probably do it via config properties. Passing the flag to binary requires to build launcher process and that can't be customized for different clusters. Ideally we should enable it for a few cluster with config properties.

Reply:
Make sense!